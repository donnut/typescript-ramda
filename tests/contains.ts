import R_contains = require('../ramda/dist/src/contains');

declare const string: string;
declare const string_array: string[];

// @dts-jest:pass:snap
R_contains(string);
// @dts-jest:pass:snap
R_contains(string, string);
// @dts-jest:pass:snap
R_contains(string, string_array);
