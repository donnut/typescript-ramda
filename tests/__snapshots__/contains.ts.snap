// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R.contains(string) 1`] = `"(list: string[] | ArrayLike<string>) => boolean"`;

exports[`R.contains(string, string) 1`] = `"boolean"`;

exports[`R.contains(string, string_array) 1`] = `"boolean"`;
